web-ui/src/main/resources/catalog/locales/en-core.json

```diff
@@ -492,5 +492,9 @@
     "warnPublishDraft": "When publishing records with workflow enabled, the status will change to 'Approve'. Are you sure you want to continue?",
     "cancelWorkingCopy": "Cancel working copy",
     "deleteWorkingCopyRecordConfirm": "Do you really want to remove the working copy '{{title}}'?",
-    "workingCopy": "Working copy"
+    "workingCopy": "Working copy",
+    "moreFacet": "more",
+    "lessFacet": "less",
+    "possibleValue": "possible values",
+    "only": "only"
```

Comment:
IMO these keys should go in `en-search.json`, but I'm not 100% sure this will work with the search in the edition module.